src/main/java/sigma/task/Task.java

```diff
@@ -0,0 +1,44 @@
+package sigma.task;
+
+/**
+ * Represents a task.
+ */
+public class Task {
+    private boolean status;
```

Comment:
I noticed you named the boolean variable "status", may I suggest an alternative name "isCompleted" instead?
https://se-education.org/guides/conventions/java/intermediate.html#comments

Reply:
Good catch, thanks for the correction!